test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: false
+
+require_relative "../helper"
+
+class TestFilter < Test::Unit::TestCase
+
+  def setup
+    @input = [
+      %w[aaa bbb ccc].join(","),
+      %w[ddd eee fff].join(","),
+      "" # Force trailing newline.
+    ].join("\n")
+  end
+
+  # Return filepath of file containing CSV data.
+  def csv_filepath(input, dirpath, option)
+    filename = "#{option}.csv"
+    filepath = File.join(dirpath, filename)
+    File.write(filepath, input)
+    filepath
+  end
+
+  # Return stdout and stderr from CLI execution.
+  def run_csv_filter(filepath, *cli_option_names)
+    top_dir = File.join(__dir__, "..", "..", "..")
+    command_line = [
+      Gem.ruby,
+      "-I",
+      File.join(top_dir, "lib"), 
+      File.join(top_dir, "bin", "csv-filter"),
+      *cli_option_names,
+      filepath,
+    ].join(" ")
```

Comment:
In general, we should use an `Array` of `String` not a `String` for `system` to avoid executing a command via shell:

```suggestion
    ]
```